### PR TITLE
mockup page for location.js

### PR DESCRIPTION
**Pull request by ameyachavan007**
Mockup for Location Page:
- fixed spacing for the entire page and add grid layout
- changed drop down menu and items style
- redesigned UI for mobile view

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,20.242.235.178,:,80,'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.235.178:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.235.178:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.235.178:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
can you add a description to this pr?

**Comment by ameyachavan007** [comment]
> can you add a description to this pr?

Done

**Comment by mikahpinegar** [comment]
i don't like the bolded text in the drop downs. can you make it default weight? 
text

**Comment by mikahpinegar** [comment]
i'm still having a lot of vertical overflow/scrolling on smaller screens but during your demo it looked like you fixed that. did you push those changes?

File: src/pages/CropSidebar/PlantHardinessZone/PlantHardinessZone.js [hunk 1]
```diff
@@ -12,43 +19,85 @@ const PlantHardinessZone = ({
...
+      },
+    },
+  };
+
   const plantHardinessZone = () => (
     <Select
-      variant="filled"
       labelId="plant-hardiness-zone-dropdown-select"
       id="plant-hardiness-zone-dropdown-select"
+      label="ZONE"
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/PlantHardinessZone/PlantHardinessZone.js:48]: should be county for MCCC and zone for NECCC/SCCC

File: src/pages/CropSidebar/PlantHardinessZone/PlantHardinessZone.js [hunk 2]
```diff
@@ -12,43 +19,85 @@ const PlantHardinessZone = ({
...
             {plantHardinessZone()}
-            {!regionShorthand
-            && (
-            <Typography variant="body2" align="center" color="error" gutterBottom>
-              {`Please Select a ${councilLabelRedux === 'Midwest Cover Crops Council' ? 'County' : 'Zone'}`}
-            </Typography>
+            {!regionShorthand && (
+              <Typography variant="body2" align="center" color="error" gutterBottom>
+                {`Please Select a ${
+                  councilLabelRedux === 'Midwest Cover Crops Council' ? 'County' : 'Zone'
```
Comment (mikahpinegar) [inline src/pages/CropSidebar/PlantHardinessZone/PlantHardinessZone.js:98]: you should use council shorthand here to avoid breaking if the label changes

Comment (ameyachavan007) [reply src/pages/CropSidebar/PlantHardinessZone/PlantHardinessZone.js]: fixed
- PlantHardinessZone now uses councilShortHand everywhere.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://,20.242.235.178,:,80,'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.235.178:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.235.178:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.235.178:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://20.242.235.178:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Pull request closed without merging**